src/ResourceManager/Compute/Commands.Compute/Extension/Diagnostics/SetAzureRmVMDiagnosticsExtension.cs

```diff
@@ -104,7 +105,7 @@ public string DiagnosticsConfigurationPath
             Mandatory = false,
             Position = 7,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The location.")]
+            HelpMessage = "The location."), LocationCompleter(new string[] { "Microsoft.Storage/storageAccounts" })]
```

Comment:
Move LocationCompleter() to [ValidateNotNullOrEmpty] like NewAzureVMCommand.cs

Reply:
No [ValidateNotNullOrEmpty] attribute here, so I moved the LocationCompleter out on its own.